addons/website_event_booth/controllers/event_booth.py

```diff
@@ -18,9 +18,12 @@ def event_booth_main(self, event, booth_category_id=False, booth_ids=False):
             raise Forbidden()
 
         booth_category_id = int(booth_category_id) if booth_category_id else False
+        view = event.booth_menu_ids.view_id
+        page = view.key
+        seo_object = request.website.get_template(page)
```

Comment:
Hello @reth-odoo 
It won't work for the multi-websites. Thus using `request.website` to get the view of the current website.
Please correct me if I am missing something.
Thanks :smile:

Reply:
@nihp-odoo I don't think it changes anything, can you have multiple views for the same event page in multi-website? If not then it would get this view anyway, but I have to say I'm not super familiar with it

So maybe :D If you have an example where you think it will mess up let me know, that'd help :+1: 
Otherwise would specify why this is needed as it's probably going to be confusing

Reply:
Hello @reth-odoo 
Yess we do have multiple views for multiple-websites!